crates/modelardb_server/tests/integration_test.rs

```diff
@@ -35,13 +35,13 @@ use datafusion::arrow::datatypes::{DataType, Field, Schema, TimeUnit::Millisecon
 use datafusion::arrow::ipc::convert;
 use datafusion::arrow::ipc::writer::{DictionaryTracker, IpcDataGenerator, IpcWriteOptions};
 use datafusion::arrow::record_batch::RecordBatch;
-use futures::stream;
+use futures::{stream, StreamExt};
 use serial_test::serial;
 use sysinfo::{Pid, PidExt, System, SystemExt};
 
 use tokio::runtime::Runtime;
 use tonic::transport::Channel;
-use tonic::Request;
+use tonic::{Request};
```

Comment:
The added `{ }` does not seem necessary and should be removed unless they are required.

Reply:
Forgot to run rustfmt. Fixed.